_posts/release/2018-03-06-accumulo-1.9.0.md

```diff
@@ -22,15 +22,47 @@ Below are resources for this release:
 
 In [ACCUMULO-4611], public API in ClientConfiguration using commons config types was deprecated to
 better support Hadoop 3 in the future. New methods were added to replace these methods which cause
-this release to be a 1.9.0 release.
+this release to be a 1.9.0 release.  These changes allow removal of commons config from Accumulo's 
+API in 2.0.0.  If using ClientConfiguration, then switching from existing constructors to the new
+static methods [create()], [fromFile()], or [fromMap()] will ensure your code works in 2.0.0.
 
 ### Performance Improvements
 
-Several peformance improvements were added in [#379], [ACCUMULO-4778], [ACCUMULO-4779] and
-[ACCUMULO-4781].
+Accumulo was profiled while running lots of concurrent small scans. During this excercise these
+performance bugs were found and fixed: [#378], [ACCUMULO-4778], [ACCUMULO-4779], [ACCUMULO-4781],
+[ACCUMULO-4782], [ACCUMULO-4788], [ACCUMULO-4789], [ACCUMULO-4790], [ACCUMULO-4797], 
+[ACCUMULO-4798], [ACCUMULO-4799], [ACCUMULO-4800], [ACCUMULO-4801], [ACCUMULO-4805], and [ACCUMULO-4809]  
```

Comment:
Could also mention the performance improvements we did for ACCUMULO-4636 (I'd forgot about these since it was so long ago haha)

Reply:
there were a few notable performance improvements... I pulled them all into this section